### DISCUSSION TITLE
Error in DateInput Component... I can't input number 31 when i am using dd/mm/yyyy

### DISCUSSION THREAD

**Discussion by phantodev** [heroui-inc/heroui#3337] [Help]
Guys... i have a weird bug here...

If i start inputing 31 number... the component change my day number to 01. :-( 

But if i input first month, For example: dd/05/yyyy and then I go back and type 31 in the day field. So 31 works.

Can someone help me?

<I18nProvider locale="pt-br">
                  <DateInput
                    label="Date Birthday"
                    labelPlacement="outside"
                    placeholderValue={new CalendarDate(1995, 11, 6)}
                    size="lg"
                  />
</I18nProvider>

**Comment by wingkwong** [comment]
ref: https://github.com/nextui-org/nextui/issues/2913
